### DISCUSSION TITLE
Error 0x80070490

### DISCUSSION THREAD

**Discussion by Pasan071** [SafeExamBrowser/seb-win-refactoring#185] [Q&A]
I tried installing SEB several times. But it always show the same error.
Can you help me?
Here is my log file

[Safe_Exam_Browser_20210704001726.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7113514/Safe_Exam_Browser_20210704001726.log)

**Comment by dbuechel** [comment]
This setup version won't work. Try using the latest installer for version 3.3.0 (https://github.com/SafeExamBrowser/seb-win-refactoring/releases/tag/3.3.0) or if you need version 3.2.0, try the updated installer for version 3.2.0.338 (https://github.com/SafeExamBrowser/seb-win-refactoring/releases/tag/3.2.0).
